### DISCUSSION TITLE
Odoo implementation

### DISCUSSION THREAD

**Discussion by magicquin** [supabase/supabase#4202] [Questions] [answered]
Hi All,

Hope that everyone is doing, I am using odoo for my front end and Postgres DB for backend. Would like to replace current Postgres DB with Supabase, as I want to be able to use all the features that come with it. Would there be any constrains or features not being available due to ORM (Odoo).

Appreciate any guidance.

**Comment by J0** [comment, marked as answer]
Hey Magicquin,

Thanks for the query, I haven't tested Odoo with Supabase personally. However, as Supabase makes use of Postgres under the hood I don't foresee any significant changes to the ORMs behavior when moving from an existing Postgres DB to Supabase.

**Comment by elvise** [comment]
> Hi All,
> 
> Hope that everyone is doing, I am using odoo for my front end and Postgres DB for backend. Would like to replace current Postgres DB with Supabase, as I want to be able to use all the features that come with it. Would there be any constrains or features not being available due to ORM (Odoo).
> 
> Appreciate any guidance.

Hi @magicquin great question! 
Do you have tested it ?

**Comment by minhng92** [comment]
Hi everyone, sorry for joining the discussion a bit late. I had a similar idea and have implemented it here: https://github.com/minhng92/odoo-supabase

At first glance, the Odoo ORM works correctly with Supabase — modules install as usual, and I’m able to view the tables and execute SQL queries directly in Supabase.
